backend/LexBoxApi/Services/ProjectService.cs

```diff
@@ -162,6 +168,7 @@ public async Task ResetProject(ResetProjectByAdminInput input)
         if (rowsAffected == 0) throw new NotFoundException($"project {input.Code} not ready for reset, either already reset or not found", nameof(Project));
         await ResetLexEntryCount(input.Code);
         await hgService.ResetRepo(input.Code);
+        await fwHeadless.DeleteRepo(await LookupProjectId(input.Code));
```

Comment:
If we end up throwing if DeleteRepo fails, I think it'd be slightly safer to put this before the ResetRepo.
If fwe delete the repo in headless and the reset in Lexbox fails for whatever reason, the system should be able to recover from that. But if only the Lexbox reset works, we're back in the weird state that is sort of the reason for this PR.